eng/scripts/Approve-Eng-Common-Sync-PRs.ps1

```diff
@@ -50,14 +55,22 @@ foreach ($repo in $repos)
   }
   
   if ($prstate.reviews.author.login -notcontains $ghloggedInUser) {
```

Comment:
I suspect we might need to also add some logic to see if we need to re-approve based on when a new commit is pushed.

Reply:
Ah, good point on the new requirement. I'll add that in.